pkg/ast/spl/spl.peg

```diff
@@ -3295,10 +3433,102 @@ EarliestOnly <- CMD_EARLIEST EQUAL earliestTime:AbsoluteTimestamp !. {
     return node, nil
 }
 
-AbsoluteTimestamp <- [0-9][0-9] '/' [0-9][0-9] '/' [0-9][0-9][0-9][0-9] ':' [0-9][0-9] ':' [0-9][0-9] ':' [0-9][0-9] {
+RelIntegerAsString <- [-+][0-9]* {
+    return string(c.text), nil
+}
```

Comment:
Did you make this new rule instead of reusing `IntegerAsString` because the `-` or `+` is required? Also do you need to change the `*` to `+` so that at least one digit is present?

Reply:
Yes -/+ is required. No, + would not be there cause you can just pass +/-.